src/objects/conf/base.py

```diff
@@ -367,8 +367,25 @@
 PROJECT_NAME = "Objects"
 SITE_TITLE = "Starting point"
 ENVIRONMENT = config("ENVIRONMENT", "")
+ENVIRONMENT_SHOWN_IN_ADMIN = True
```

Comment:
Why is this setting needed? I don't see it being used anywhere

Reply:
It shows the `ENVIRONMENT` setting for the admin login page (and whenever messages are shown, [see base_site.htm](https://github.com/maykinmedia/objects-api/blob/master/src/objects/templates/admin/base_site.html#L39)l):

![image](https://github.com/maykinmedia/objects-api/assets/5655384/bcf17dcf-d59f-448d-b42d-accfd8e5ce58)

Reply:
oh, ok!